node/test/toolrunnertests.ts

```diff
@@ -294,16 +294,18 @@ describe('Toolrunner Tests', function () {
                 });
         }
         else {
-            var ls = tl.tool(tl.which('ls', true));
-            ls.arg('-j');
-
+            var bash = tl.tool(tl.which('bash', true));
```

Comment:
`ls -j` returns 1 on OSX, but 2 on Ubuntu 14.

I just switched to `bash -c` where we have control over stdout, stderr, and exit code.